includes/namespace.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Plugin initialization file.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories;
+
+/**
+ * Handles plugin activation.
+ *
+ * Throws an error if the site is running on PHP < 5.6
+ *
+ * @since 1.0.0
+ *
+ * @param bool $network_wide Whether to activate network-wide.
+ */
+function activate( $network_wide ) {
```

Comment:
Why are these hooks needed right now?

Reply:
Mostly because of the PHP version check, which previously did nothing (just stopped script execution). I took this as an opportunity to fix that and move that logic to this new file here. Not really related to the change here I know, but a step toward the refactoring.

I can also split it out if wanted